voltixApp/VoltixApp/Views/Swap/SwapCryptoDetailsView.swift

```diff
@@ -47,7 +45,7 @@ struct SwapCryptoDetailsView: View {
     var fromCoinField: some View {
         VStack(spacing: 8) {
             getTitle(for: "from")
-            TokenSelectorDropdown(title: "Ethereum", imageName: "eth", amount: "")
+//            TokenSelectorDropdown(tx: tx, coinViewModel: <#CoinViewModel#>, group: group)
```

Comment:
I am not sure what happens here , if it is not needed , then remove it , otherwise fix it